# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/git/fix-docbook2texi-path.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- git-2.45.1/Documentation/Makefile.vanilla 2024-05-19 05:45:43.271000000 +0200
+++ git-2.45.1/Documentation/Makefile 2024-05-19 05:45:50.581000000 +0200
@@ -168,7 +168,7 @@
 
 MAKEINFO = makeinfo
 INSTALL_INFO = install-info
-DOCBOOK2X_TEXI = docbook2x-texi
+DOCBOOK2X_TEXI = docbook2texi
 DBLATEX = dblatex
 ASCIIDOC_DBLATEX_DIR = /etc/asciidoc/dblatex
 DBLATEX_COMMON = -p $(ASCIIDOC_DBLATEX_DIR)/asciidoc-dblatex.xsl -s $(ASCIIDOC_DBLATEX_DIR)/asciidoc-dblatex.sty
